index.d.ts

```diff
@@ -126,14 +132,53 @@ interface ESMSInitOptions {
 
   /**
    * #### Revoke Blob URLs
-   * 
+   *
    * Set to *true* to cleanup blob URLs from memory after execution.
    * Can cost some compute time for large loads.
-   * 
+   *
    */
   revokeBlobURLs: boolean;
 }
 
+interface ImportMap {
+  imports: Record<string, string>;
+  scopes: Record<string, Record<string, string>>;
+}
+
+type ResolveFn = (id: string, parentURL?: string) => string;
+
+interface RegistryLoad {
+  /** URL */
+  u: string;
+  /** Response URL */
+  r: string | undefined;
+  /** Fetch Promise */
+  f: Promise<RegistryLoad> | undefined;
+  /** Source */
+  S: string | undefined;
+  /** Link Promise */
+  L: Promise<void> | undefined;
+  /** Analysis */
+  a: ReturnType<typeof parse> | undefined;
+  /** Deps */
+  d:
+    | (RegistryLoad | Pick<RegistryLoad, "r" | "b"> | Pick<RegistryLoad, "b">)[]
+    | undefined;
+  /** Blob URL */
+  b: string | undefined;
+  /** Shell URL */
+  s: string | undefined;
+  /** Needs shim */
+  n: boolean;
+  /**
+   * Module type
+   * @deprecated This field is never set.
+   */
+  t: null;
```

Comment:
I guess this property could be removed completely because I don't see it being used (there is another `t` property being returned by the fetch functionality, but that object does not appear to ever overlap with the one set in `registry`).